.gitignore

```diff
@@ -9,6 +9,9 @@
 /docs-build
 /plugins
 
+# local dev build output
+electron-dev/
```

Comment:
Everything runs out of this dir for development; in this case, it's basically a copy of `public/`.

`www` is left alone as the existing build output dir.